### ISSUE TITLE
Feature request: add infinite-order Mordell-Weil generators as columns in Elliptic Curve search results

### ISSUE THREAD

**Issue by alonamit1729** [LMFDB/lmfdb#5836]
When searching for elliptic curves it is currently not possible to include the Mordell-Weil generators as columns in the search results. It may be cumbersome to do so for curves of high rank, but for rank 1 (and possibly 2, 3) it would be extremely valuable to include them.

**Comment by edgarcosta** [comment]
This is doable. However, it is not straightforward, as MW generators live in a different table from the search table.

**Comment by roed314** [comment]
The main problem is with downloading, if we're getting the data from another table.

**Comment by edgarcosta** [comment]
I agree, another not straightforward point :)

**Comment by alonamit1729** [comment]
I should have mentioned that downloading is my main motivation :) I hope a `join` is possible!

**Comment by roed314** [comment]
Ah.  Yeah, I didn't get downloading to work.  If you set up the LMFDB locally, you can do the following:
```
from lmfdb import db
D = {rec["lmfdb_label"]:rec["gens"] for rec in db.ec_mwbsd.search({}, ["lmfdb_label", "gens"])}
```
This gives you a dictionary with all the generators for all the curves in the LMFDB, and takes about 3.5 minutes to run.

**Comment by JohnCremona** [comment]
In case it is helpful, the repositories https://github.com/JohnCremona/ecdata and https://github.com/JohnCremona/ecnf-data have the raw data as well as Sage code for reading the files.  Also, the web page https://johncremona.github.io/ecdata/ (which long predates the LMFDB) allows you to download all the generators in text files called allgens.* split into conductor ranges of 10000.

Of course it really should be true that anything you can do with these files you can also do with the LMFDB.

**Comment by alonamit1729** [comment]
Thank you!!

**Comment by roed314** [comment]
This was fixed in #6259, though it will be slow if there are a lot of search results (I haven't tested, so I would be curious what the actual limit is before you hit a 30 second timeout).
